### DISCUSSION TITLE
[V4] Postcss: Top-level selectors may not contain the parent selector "&".

### DISCUSSION THREAD

**Discussion by Matheun** [tailwindlabs/tailwindcss#14178] [Help]
**Tailwind version:**
v4.0.0-alpha.19

**Framework/ build tools:**
Nuxt 3.12.3, postcss, sass 1.77.8

**Relevant Nuxt config:**
```ts
import { fileURLToPath } from "node:url";
import { dirname, join } from "node:path";

const currentDir = dirname(fileURLToPath(import.meta.url));

export default defineNuxtConfig({
    typescript: {
        includeWorkspace: true,
    },

    css: [join(currentDir, "./assets/scss/vendors/tailwind.scss")],

    postcss: {
        plugins: {
            "@tailwindcss/postcss": {},
            "postcss-custom-media": {},
        },
    },
    ...
});

```

**Issue:**
The following error gets thrown:
```bash
Error: Top-level selectors may not contain the parent selector "&".
```

The code snippit thats causing this issue is the following:
```scss
@utility button {
    ...
    @supports selector(:focus-visible) {
        &:focus-visible { // <-- this line
            ...
        }
    }
}
```

This issue still exists when refactoring the code like so:
```scss
@utility button {
    ...
    &:focus-visible { // <-- this line
        ...
    }
}
```

**Comment by adamwathan** [comment]
Hey! The error you are seeing comes from Sass:

https://sass-lang.com/playground/#eJwzNHQoLcnMySypVEgqLSnJz1Oo5lJQULNKy08uLdYtyyzOTMpJVahWAAoqKCTn5+QXWSkk5ZSmWgMFarlqAZqBE+I=

Is there a reason you need to use Sass? Tailwind CSS v4 and Sass are both sort of preprocessors so if you try to use them together there will be conflicts like this.
